lib/management.js

```diff
@@ -111,11 +115,23 @@ function patchConfiguration(instanceId, newConf, log, cb) {
                         };
                     }
                     break;
-                case 'location-do-spaces-v1':
+                case 'location-scality-ring-s3-v1':
+                    pathStyle = true; // fallthrough
```

Comment:
I would recommend not relying on fallthrough capability of `switch`, instead using a map of properties per location name (`{ 'location-scality-bleh': { pathStyle: true, ...} }`) would be more readable and less error-prone.